cmake/projects/gstreamer/hunter.cmake

```diff
@@ -36,7 +36,7 @@ hunter_cacheable(gstreamer)
 hunter_download(
     PACKAGE_NAME gstreamer
     PACKAGE_INTERNAL_DEPS_ID "4"
-    PACKAGE_UNRELOCATABLE_TEXT_FILES
+    PACKAGE_UNRELOCATABLE_TEXT_FILES_OPTIONAL
```

Comment:
I don't think they **all** should be optional; some files should present always (?).

Reply:
Agreed for gstreamer itself, will fix that. For the plugins I guess most of not all will be optional.